aws/resource_aws_route53_record.go

```diff
@@ -974,6 +977,5 @@ func parseRecordId(id string) [4]string {
 			}
 		}
 	}
-	recName = strings.TrimSuffix(recName, ".")
```

Comment:
I think we can keep this here now as well (and the previous unit test case) 👍

Reply:
yess just discovered this one! interesting enough, i think our tests don't seem to cover a case where it would fail without this line back in b/c in TC the tests all just passed ...

Reply:
We would need to implement an acceptance test with a custom `ImportStateIDFunc` since the `ImportState` testing pulls in the ID with the trimmed value. Maybe good to have, but doesn't need to happen right now. 😄

Reply:
Hmm I think the `TestAccAWSRoute53Record_basic_trailingPeriodAndZoneID` I most recently added _should hopefully_ account for catching the importstateverify error ... atleast it fails when the above code is omitted :)